pytests/conftest.py

```diff
@@ -220,7 +220,7 @@ def _run(self, cmd, retvalonly=False):
         stdout = out.decode().strip()
         stderr = err.decode().strip()
         retval = process.returncode
-        capture = re.match(r'^Error\((\d+)\) :', stderr)
+        capture = re.search(r'Error\((\d+)\) :', stderr)
```

Comment:
Regex should be the same: `r'^Error\((\d+)\) :'` as we still expect Error to be in the beginning of a newline.